server/testing-queue.conf

```diff
@@ -0,0 +1,17 @@
+include classpath("application.conf")
```

Comment:
What dis?

Reply:
Something that shouldn't have been added. Removed.